docs/api-server/README.md

```diff
@@ -1,6 +1,19 @@
 # Kops HTTP API Server
 
-### Notes:
+### Building
 
+To build the API container run the following
 
- - https://github.com/kubernetes/kubernetes/pull/40803
\ No newline at end of file
+Note: This is a MAJOR clockwork, and a lot of assumptions and hardcoding exist here for now.
```

Comment:
As in "clockwork orange" 

"This PR is a clockwork" ~> "This PR is a work in progress"